api.yaml

```diff
@@ -46,31 +46,21 @@ info:
     name: Apache 2.0
     url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
 paths:
-  /healthcheck:
```

Comment:
question: even though this api.yaml isn't really functional, why get rid of the healthcheck?

Reply:
added it back